base58/Cargo.toml

```diff
@@ -23,6 +23,7 @@ internals = { package = "bitcoin-internals", version = "0.4.0" }
 
 [dev-dependencies]
 hex = { package = "hex-conservative", version = "0.3.0", default-features = false, features = ["alloc"] }
+internals = { package = "bitcoin-internals", version = "0.4.0", features = ["hex"] }
```

Comment:
Looks like this needs `hex_lit` dependency instead.

Reply:
Woops, I tried that before I worked out to throw `.to_vec()` everywhere. Done as suggested.